### PR TITLE
Prepare for v0.13.0 release

### PR DESCRIPTION
**Pull request by TommyCpp**
- Prepare CHANGELOGs to include PRs from #433. 
- Fix docs that still use tracer provider guard.
- Update the protobuf generated file version.
- Added missing features in `examples/external-otlp-tonic-tokio`.

### CODE REVIEW

File: opentelemetry-prometheus/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v0.6.0
+### Added
+- Add sanitization of prometheus label names #462
+
+### Changed
+- Update to opentelemetry v0.13.0
+- Update `prometheus` dependency to v0.11 #435
```
Comment (jtescher) [inline opentelemetry-prometheus/CHANGELOG.md:9]: think this already happened in 0.5.0?

File: opentelemetry/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## [v0.13.0](https://github.com/open-telemetry/opentelemetry-rust/compare/v0.12.0...v0.13.0)
+
+## Changed
+
+- Changed the default branch to `main` #439
```
Comment (jtescher) [inline opentelemetry/CHANGELOG.md:7]: Not sure we need this in the changelog, probably more of a housecleaning PR

**Review by jtescher — CHANGES_REQUESTED** [review]
Thanks @TommyCpp, just a couple comments

**Comment by djc** [comment]
I feel like it might be better to try and get #467 and #469 in before we release, since they seem meaningful improvements with relatively large API impact?

File: examples/external-otlp-tonic-tokio/Cargo.toml [hunk 3]
```diff
@@ -5,16 +5,8 @@ edition = "2018"
...
-    "metrics",
-    "serialize"
-] }
-opentelemetry-otlp = { path = "../../opentelemetry-otlp", features = [
-    "tonic",
-    "tls",
-    "tls-roots",
-] }
+opentelemetry = { path = "../../opentelemetry", features = ["rt-tokio", "metrics", "serialize"] }
+opentelemetry-otlp = { path = "../../opentelemetry-otlp", features = ["tonic", "tls", "tls-roots", "async"] }
```
Comment (TommyCpp) [inline examples/external-otlp-tonic-tokio/Cargo.toml:9]: Need to remove the `async` feature once the #483 removed

**Review by jtescher — APPROVED** [review]
Looks good, thanks @TommyCpp

**Pull request merged**